content/rs/references/cli-utilities/crdb-cli/task/status.md

```diff
@@ -19,20 +19,18 @@ crdb-cli task status --task-id <task_id>
 | Parameter           | Value  | Description                         |
 |---------------------|--------|-------------------------------------|
 | task-id \<task_id\>  | string | An Active-Active database task ID (required) |
+| verbose              | N/A    | Returns detailed information when specified |
+| no-verbose           | N/A    | Returns limited information when specified |
 
 ### Returns
 
 Returns the status of an Active-Active database task.
 
+The `--verbose` and `--no-verbose` options are mutually incompatible; specify one or the other.
```

Comment:
It might be a good idea to move this part under the parameters table instead of in the returns section.

Reply:
Fixed.